tests/integration/cli.py

```diff
@@ -0,0 +1,24 @@
+import ast
+import pytest
+import subprocess
+
+PROJECT1 = 'foo'
+
+
+class TestCLI:
```

Comment:
This doesn't need to be in a class; pytest will pick up top-level functions called `test_*`. Do that unless there's a reason to introduce structure via a class.